src/components/Tour/Tour.vue

```diff
@@ -98,80 +102,53 @@ export default {
     callbacks: {
       type: Object,
     },
+    steps: {
+      default: () => [],
+      type: Array,
+    },
   },
-    data () {
+  data () {
     return {
       tour: {
-        name:'app-list',
-        steps: [   
-          'app-list',
-          'low-code',
-          'crm',
-          'reporter',
-          'workflow',
-          'profile',
-          ].map(step=>{
-            return {
-                    name: step,
-                    target: `[data-v-onboarding="${step}"]`,
-                    header: {
-                      title: this.$t(`steps.${step}.title`),
-                    },
-                    content: this.$t(`steps.${step}.content`),
-                  }
-          }),
+        name: 'app-list',
+        steps: this.steps.map(step => {
+          return {
+            name: step,
+            target: `[data-v-onboarding="${step}"]`,
+            header: {
+              title: this.$t(`steps.${step}.title`),
+            },
+            content: this.$t(`steps.${step}.content`),
+          }
+        }),
       },
     }
   },
   computed: {
-      labels () {
-        return this.tour.labels
-      },
+    labels () {
+      return this.tour.labels
     },
+  },
   methods: {
     start () {
-      if (JSON.parse(localStorage.getItem('corteza.tour'))) {
+      if (JSON.parse(localStorage.getItem('corteza.tour')) && this.$tours[this.tour.name].steps.length) {
```

Comment:
Can't you just check the `this.tour.steps.length` ? Since you will initiate a new tour for every webapp. the `this` object refers to this tour right? Or am I missing something?